pkg/cmd/scan.go

```diff
@@ -323,6 +345,26 @@ func scanRun(opts *pkg.ScanOptions) error {
 	return nil
 }
 
+func parseScopeFlag(scope []string) ([]string, error) {
+
+	scopeRegex := `projects/\S*$|folders/\d*$|organizations/\S*$`
```

Comment:
I've crafted this regex. The reason I did not put more constraints for organizations was to make future-proof, in case GCP will switch to a different style of ID. But thinking about it, I did put the constraint on the folder scope, I'll add it also to the org